RUFAS/routines/animal/ration/animal_requirements.py

```diff
@@ -522,12 +522,12 @@ def calc_rqmts(
                 parity,
             )
         else:
-            energy_and_nutrient_calculation_method_error = f"energy and nutrient calculation method \
-                {AnimalBase.config['energy_and_nutrient_calculation_method']}\
+            nutrient_standard_error = f"Nutrient Standard \
+                {AnimalBase.config['nutrient_standard']}\
```

Comment:
```suggestion
                {AnimalBase.config['nutrient_standard']} \
```
I think this ensures there is a space between the formatted input and "not supported", but not 100% sure.

Reply:
Done